client/src/catalog-element.html

```diff
@@ -403,6 +423,22 @@ <h2 title>[[subElement]]</h2>
         this._routeChanged(this.route, this.visible);
       },
 
+      _showDemo: function(e) {
```

Comment:
We shouldn't use this approach, its pretty confusing and demos aren't linkable anymore.

Instead let's use the routing that's already set up. In `_routeChanged()` we just need to check `this.demo` and open/close the dialog based on that.

Closing the dialog should also result in a route change.